### ISSUE TITLE
Missing / Broken parts

### ISSUE THREAD

**Issue by michaelconnor** [rhizomedotorg/The-Art-Happens-Here-Exhibition-Documentation#1]
– [ ] Short ethernet cable
- [ ] Lialina slide projectors are a bit wonk

**Comment by Apophenia** [comment]
Is this an ongoing list of things we're going to source/mail to Denver ourselves?

**Comment by michaelconnor** [comment]
Yes @Apophenia – we need to do a call with Denver to talk through

**Comment by michaelconnor** [comment]
- [ ] additional blank SD cards?
- [ ] USB to AC plugs?
- [ ] micro USB cables?
- [ ] VGA/HDMI adaptors?
